Ember Times 208 (#1308)
ember-learn/ember-blog: content/the-ember-times-issue-208.md

---

## [🇪🇸 EmberFest 2023 videos](https://www.youtube.com/@emberfest)

On the 21st and 22nd of September about 150 Emberistas got together for [EmberFest 2023](https://emberfest.eu) in Madrid. The great variaty of talks is now available on [YouTube](https://www.youtube.com/@emberfest). Checkout all amazing speakers, both those on the [schedule](https://emberfest.eu/schedule/day/1/), as well as a suprise [Q&A](https://www.youtube.com/watch?v=oY289Gk6NcY) with [Ed Faulkner (@ef4)](https://github.com/ef4) and the [lightning talks](https://www.youtube.com/watch?v=VVvRxAOBCVQ).

---

## [🇪🇺 Ember Europe: Update on Embroider initiative](https://www.youtube.com/watch?v=gPMzrQzryZI)

[Chris Manson (@mansona)](https://github.com/mansona) provided an update on the [Embroider Initiative](https://mainmatter.com/embroider-initiative/)'s progress during his talk at Ember Europe’s final 2023 Meetup. 

He addressed the [Vite](https://vitejs.dev) support and demonstrated an Ember app on Vite, explaining the importance of using ESM-compatible dependencies. He also discussed the challenges faced during the integration process, such as add-ons not being fully converted to ESM. 

> Chris introduced the upcoming [unstable Embroider release](https://github.com/embroider-build/embroider/releases) and encouraged users to remain optimistic despite encountering issues due to cache busting by showcasing debugging techniques and the advantages of Vite's network tab.

Reviewer: ```suggestion
Chris introduced what will be in the [next major release of Embroider](https://github.com/embroider-build/embroider/pull/1701) and encouraged users to remain optimistic despite encountering issues due to cache busting by showcasing debugging techniques and the advantages of Vite's network tab.
```
